.github/workflows/maintainer_merge.yml

```diff
@@ -0,0 +1,54 @@
+name: Maintainer merge
+
+on:
+  issue_comment:
+    types: [created, edited]
+  pull_request_review:
+    types: [submitted]
```

Comment:
I think this needs to be split into a separate workflow file; note that `add_label_from_review.yml` uses `github.event.review.body` rather than `github.event.comment.body`.